firestore/integration_test_internal/src/android/firestore_integration_test_android.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_TESTS_ANDROID_FIRESTORE_INTEGRATION_TEST_ANDROID_H_
 #define FIREBASE_FIRESTORE_CLIENT_CPP_SRC_TESTS_ANDROID_FIRESTORE_INTEGRATION_TEST_ANDROID_H_
```

Comment:
Lint: `Lines should be <= 80 characters long `<hidden value="cpplint-file-comment"></hidden>